scripts/debug-publish-to-unity-package.bat

```diff
@@ -1,30 +1,42 @@
-@ECHO OFF
+
+
+SET SCRIPT_DIR=%~dp0
 
 echo Building project...
-pushd src/ChainSafe.Gaming.Unity
+pushd "%SCRIPT_DIR%\..\src\ChainSafe.Gaming.Unity"
 
 del obj /F /Q
 del bin /F /Q
+dotnet restore
 dotnet publish -c debug -f netstandard2.1 /property:Unity=true
 if %errorlevel% neq 0 exit /b %errorlevel%
 
 echo Restoring non-Unity packages...
 
-pushd ..\..
-dotnet restore
-popd
-
 echo Moving files to Unity package...
 
-pushd bin\debug\netstandard2.1\publish
+pushd bin\release\netstandard2.1\publish
```

Comment:
This is `debug-publish-to-unity-package.bat` probably this one should not be changed.